charts/datadog/requirements.yaml

```diff
@@ -5,7 +5,3 @@ dependencies:
     condition: clusterAgent.metricsProvider.useDatadogMetrics
     tags:
     - install-crds
-  - name: kube-state-metrics
-    version: 2.13.2
-    repository: https://prometheus-community.github.io/helm-charts
-    condition: datadog.kubeStateMetricsEnabled
```

Comment:
the idea was to keep optional the installation on the `kube-state-metrics` chart dep.